### PR TITLE
Model equations with illustration in README

### PR DESCRIPTION
**Pull request by Peter9192**
This PR adds the model equations for forward and backward tracking to the README. I also made an illustration :-)

Please check it carefully. To preview a rendered version: https://github.com/WAM2layers/WAM2layers/tree/bdec77a44e4606b8e1a8c77133d6c1c16a1169ee

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,31 @@ code originally developed by Ruud van der Ent. It can be used to determine where
...
+            &+F_{ns,n}\ast s_{t,n}-F_{ns,s}\ast s_t &             &+F_{ns,s}\ast s_{t,s}-F_{ns,n}\ast s_t \\
+            &+F_{sn,s}\ast s_{t,s}-F_{sn,n}\ast s_t &             &+F_{sn,n}\ast s_{t,n}-F_{sn,s}\ast s_t \\
+            &+F_{ul,u}\ast s_{t,u}-F_{ul,l}\ast s_t &             &+F_{ul,l}\ast s_{t,l}-F_{ul,u}\ast s_t \\
+            &+F_{lu,l}\ast s_{t,l}-F_{lu,u}\ast s_t &             &+F_{lu,u}\ast s_{t,u}-F_{lu,l}\ast s_t \\
+            &+E-P                                   &             &-E+P
+\end{align}
+$$
+
+where $F$ are (total) moisture fluxes and $s$ represents the amount of tracked relative to total moisture in the grid cells. 
+The up/down fluxes are not illustrated but they follow the same systematic as the north/south and east/west fluxes.
```
Comment (ruudvdent) [inline README.md:25]: what is meant by this line? The F_vert fluxes or the upper/lower layer fluxes?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,31 @@ code originally developed by Ruud van der Ent. It can be used to determine where
...
+            &+F_{ul,u}\ast s_{t,u}-F_{ul,l}\ast s_t &             &+F_{ul,l}\ast s_{t,l}-F_{ul,u}\ast s_t \\
+            &+F_{lu,l}\ast s_{t,l}-F_{lu,u}\ast s_t &             &+F_{lu,u}\ast s_{t,u}-F_{lu,l}\ast s_t \\
+            &+E-P                                   &             &-E+P
+\end{align}
+$$
+
+where $F$ are (total) moisture fluxes and $s$ represents the amount of tracked relative to total moisture in the grid cells. 
+The up/down fluxes are not illustrated but they follow the same systematic as the north/south and east/west fluxes. 
+Note that all fluxes are by definition positive; this is needed because moisture flux is scaled with the relative amount of tracked moisture 
+in the grid cells where it originates from.
```
Comment (ruudvdent) [inline README.md:27]: no need to start new line

**Review by ruudvdent — APPROVED** [review]
excellent figure, please note the small comments to the text

File: README.md (hunk 1 above)
Comment (Peter9192) [reply README.md]: The vertical fluxes. I reworded it.

**Comment by Peter9192** [comment]
Thanks for reviewing

**Pull request merged**